docs/reference/commands/disk.rst

```diff
@@ -33,13 +33,27 @@ Global flags:
 ``add``
 -------
 
-Adds a disk to the cluster, alongside optional devices for write-ahead logging and database management.
+Adds a new Ceph disk (OSD) to the cluster, alongside optional devices for write-ahead logging and database management.
+
+The command takes a parameter SPEC which is either a path to a block device such as /dev/sdb or a specification for one or more loop files.
```

Comment:
Is "loop file" a thing? Not "loopback file" or "loopback device"?

So it's written "a block device" but "one or more loopback devices". Should they not both be either singular or multiple?

Reply:
We're not actually using a loop device (i.e. the things `losetup` configures) since Ceph has the ability to directly writing to a backing file. "Loop file" is our shorthand for that (but happy to take suggestions on wording that better). A "loopback device" refers to the `lo` device in networking, so that doesn't really fit here. 

In  the current implementation we indeed have the ability to either add a single (real) block device or several OSDs backed by files

Reply:
Okay,let's go with this.